### PR TITLE
Added AWS helm deployment example.

### PR DESCRIPTION
**Pull request by xueshanf**
See #334.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,14 +22,21 @@ You can use the chart in this repository to install Monocular in your cluster.
 - [Helm and Tiller installed](https://github.com/kubernetes/helm/blob/master/docs/quickstart.md)
 - [Nginx Ingress controller](https://kubeapps.com/charts/stable/nginx-ingress)
   - Install with Helm: `helm install stable/nginx-ingress`
+
   - **Minikube/Kubeadm**: `helm install stable/nginx-ingress --set controller.hostNetwork=true`
+  
+  - **AWS**: For testing purpose, the install example below will set ELB security rule to allow connections from $MY_IP only.
```
Comment (prydonius) [inline README.md]: By default, does it not allow connections from anywhere?

Comment (xueshanf) [inline README.md]: It does allow connections from anywhere by default. By setting loadBalancerSourceRanges in this example,  we can limit the exposure before one has a chance to setup authentication.

Comment (prydonius) [inline README.md]: While that's valid, I'm not sure it makes sense at this point in the documentation. Also, perhaps loadBalancerSourceRanges is not AWS-specific and can be used on other platforms.

Comment (xueshanf) [inline README.md]: You are right. I don't have other providers environment setup to test, but I think it is safe to make it a generic helm command line override.

Comment (xueshanf) [inline README.md]: Or for consistency, not use the setting here but mention it in elsewhere in the doc?

Comment (prydonius) [inline README.md]: I think we could move this to the chart README under a section about running Monocular in production

**Comment by prydonius** [comment]
@xueshanf are you okay with moving the loadBalancerSourceRanges to the chart README?

**Comment by xueshanf** [comment]
On Wed, Aug 23, 2017 at 5:43 AM, Adnan Abdulhussein <
notifications@github.com> wrote:

> @xueshanf <https://github.com/xueshanf> are you okay with moving the
> loadBalancerSourceRanges to the chart README?
>

Yes! I am just not sure which README file to edit.. if you have time to
take care of this, go for it, otherwise, let me know!

Thanks!

Xueshan

**Comment by prydonius** [comment]
@xueshanf I won't have time this week, but can get to it next week if you can't :)

**Comment by prydonius** [comment]
Hey @xueshanf, I wasn't able to get to this, let me know if you can update this

**Comment by xueshanf** [comment]
> On Sep 18, 2017, at 6:08 AM, Adnan Abdulhussein <notifications@github.com> wrote:
> 
> Hey @xueshanf, I wasn't able to get to this, let me know if you can update this
> 

Let me know where it should belong and I will update! Thanks!

Xueshan

**Comment by prydonius** [comment]
@xueshanf sorry, missed your reply. I think a new section in the [Monocular chart README](https://github.com/kubernetes-helm/monocular/tree/master/deployment/monocular) about running in production would be a good place to put this.

**Comment by xueshanf** [comment]
@prydonius Now I think about it, The IP range restriction that I mentioned in AWS deployment is not even for production. It's just for test with IP restriction. I am not sure if I should start a "Running in Production" section.  Thoughts?

**Comment by prydonius** [comment]
hmm @xueshanf, good point. I actually think this option is a bit confusing, how about we remove `controller.service.loadBalancerSourceRanges={${MY_IP}}` and the explanation and just have the `publishService` option?

**Comment by prydonius** [comment]
Closing for now since this has become stale. @xueshanf feel free to open a new updated PR!

**Comment by thelinuxfoundation** [comment]
Thank you for your pull request. Before we can look at your contribution, we need to ensure all contributors are covered by a Contributor License Agreement.

After the following items are addressed, please respond with a new comment here, and the automated system will re-verify.

* User @xueshanf isn't covered by a CLA. They will need to complete the form at https://identity.linuxfoundation.org/projects/cncf

_Regards,
CLA GitHub bot_

**Pull request closed without merging**